src/algorithms/calorimetry/ImagingTopoClusterConfig.h

```diff
@@ -22,7 +22,9 @@ struct ImagingTopoClusterConfig {
   // maximum distance of global (x, y) to be considered as neighbors at different layers (if layerMode==xy)
   std::vector<std::variant<std::string, double>> layerDistXY = {1.0 * dd4hep::mm, 1.0 * dd4hep::mm};
   // determines how neighbors are determined for hits in different layers (using either eta and phi, or x and y)
-  enum ELayerMode { etaphi = 0, xy = 1 } layerMode = etaphi;
+  enum class ELayerMode { etaphi = 0, xy = 1, phiz = 2 };
+  ELayerMode sameLayerMode = ELayerMode::xy; // for ldiff =0
+  ELayerMode diffLayerMode = ELayerMode::xy; // for ldiff <= neighbourLayersRange
```

Comment:
Would it make sense to have a naming consistent with parameters:
```suggestion
  ELayerMode localMode = ELayerMode::xy; // for ldiff =0
  ELayerMode layerMode = ELayerMode::xy; // for ldiff <= neighbourLayersRange
```

Reply:
Yes, this is much better.